src/main/java/org/terasology/oreGeneration/generation/BaseOreGen.java

```diff
@@ -27,8 +27,8 @@ public StructureDefinition createStructureDefinition(GeneratingRegion region) {
         if (!isInRange(region)) {
             return null;
         }
-        Vector3i regionSize = region.getRegion().size();
-        float scaleFactor = regionSize.getY() / 10f;
+        Vector3i regionSize = region.getRegion().getSize(new Vector3i());
+        float scaleFactor = regionSize.y() / 10f;
```

Comment:
Looks like we only need the size along the Y axis:
```suggestion
        float scaleFactor = region.getRegion().getSizeY() / 10f;
```